guides/tutorial/start-tutorial.md

```diff
@@ -37,12 +37,47 @@ end
 
 ## Create a user manager
 
-We'll need something to authenticate. How Users are created and what they can do is outside the scope of this tutorial. If you already have a user model you can skip this part.
+We'll need something to authenticate. How Users are created and what they can do is outside the scope of this tutorial. If you already have a User model you can skip this part (but see the note after the command).
 
 ```sh
 $ mix phx.gen.context UserManager User users username:string password:string
 ```
 
+Note: if you already have a User model, you need to implement some of the
+functions that mix generates for the UserManager context. mix generates the
```

Comment:
I think practically its phoenix that generates these :)

Reply:
Okay.  I wanted to make sure the reader knew exactly what command I was referring to.   How about I change `mix` to phx.gen.context`?  And...how do I do that?  Do I make a new pull request with another branch name with just that change?

Reply:
I completely agree that `phx.gen.context` is better. To fix it simply just make a new commit on your current branch and push it. Then it will show up here and will do a Squash and merge to make it into one commit in the end :)

Reply:
Okay, I'll do that.  I was reviewing some of the text and doing some additional editing, and I occasionally found it confusing  when the tutorial refers to a `resource`.  In Guardian speak, a resource is a User.  However, the tutorial also uses the word resource in the web sense of the word, e.g. a web page.   I thought it would be clearer if I changed the word `resource` to `User` when appropriate, and to `web page` when that was the intended meaning.   But, in the bolded text below I cannot figure out in what sense the word resource is being used:  

> Routes
> 
> Okay. So the controller and views are not strictly part of Guardian but we need some way to interact with Guardian. From here, the only thing left for us to do is to wire up our router.
> 
> # Our AuthMeUserManager.Pipeline module implements "maybe" authenticated. We'll use an `:ensure_auth` pipeline below for when we need to make sure someone is logged in.
> ```
> pipeline :auth do
>   plug AuthMe.UserManager.Pipeline
> end
> 
> # The ensure_auth pipeline will fail if no one is logged in:
> pipeline :ensure_auth do
>   plug Guardian.Plug.EnsureAuthenticated
> end
> 
> # Maybe logged in routes:
> scope "/", AuthMeWeb do
>   pipe_through [:browser, :auth]
> 
>   get "/", PageController, :index
> 
>   get "/login", SessionController, :new
>   post "/login", SessionController, :login
>   get "/logout", SessionController, :logout
> end
> 
> # Definitely logged in routes:
> scope "/", AuthMeWeb do
>   pipe_through [:browser, :auth, :ensure_auth]
> 
>   get "/protected", PageController, :protected
> end
> 
> ```
> There's a little bit happening here:
> 
> We created a Phoenix pipeline that just delegates to our Guardian pipeline to login someone if we find a token in the session info or Authorization header. This does not restrict access.
> We decide what type of access the User has with the :auth and :ensure_auth phoenix pipelines. In order to protect our protected route, we use :ensure_auth.
> Note that you must use the :auth pipeline before the :ensure_auth pipeline to make sure that we have fetched and verified the token. ??? **We're also loading the resource but that is not required for :ensure_auth**.???

Reply:
In that sense its a user resource that is loaded by EnsureAuthenticated